src/lib/event.ts

```diff
@@ -193,18 +197,26 @@ const getAllEvents = async (): Promise<Event> => {
 const getEvent = async (pid: number): Promise<EventWithCid> => {
   const event = await getObject(`${listKey}:pid:${pid}`);
   const cid = await getCidByPid(event.pid);
+  const fixedEvent = fixEvent(event);
 
   return {
-    ...fixEvent(event),
+    ...fixedEvent,
+    responsesCount: await getResponsesCount(+event.pid),
     cid,
   };
 };
 
 const getEventsEndingAfter = async (endDate: number): Promise<Event[]> => {
   const keys = await getSortedSetRangeByScore(listByEndKey, 0, -1, endDate, +Infinity);
-  const events = (await getObjects(keys)).filter(Boolean);
+  const events: JsonEvent[] = (await getObjects(keys)).filter(Boolean);
+  const pids = events.map(event => +event.pid);
```

Comment:
If you fix the events first, `pid` should be a number and you won't need to convert with `+`